save-backend/src/test/kotlin/org/cqfn/save/backend/DownloadFilesTest.kt

```diff
@@ -58,19 +59,21 @@ import kotlin.io.path.writeLines
 class DownloadFilesTest {
     @Autowired
     lateinit var webTestClient: WebTestClient
+    @Autowired private lateinit var fileSystemRepository: FileSystemRepository
```

Comment:
Dose it done intentionally?

```suggestion
    @Autowired
    private lateinit var fileSystemRepository: FileSystemRepository
```

Reply:
Maybe, we should support unified style for annotations in diktat? Currently we are only checking several annotations, not single ones.

Reply:
Good idea